src/routes/v2/project_creation.rs

```diff
@@ -233,6 +235,11 @@ struct ProjectCreateData {
     #[serde(default = "default_requested_status")]
     /// The status of the mod to be set once it is approved
     pub requested_status: ProjectStatus,
+
+    // Associations to uploaded images in body/description
+    #[validate(length(max = 10))]
+    #[serde(default)]
+    pub uploaded_images: Vec<ImageId>,
```

Comment:
not sure the point of this either, since image uploading is a separate route

Reply:
resolving with discord convo